src/Collectors/FrameworkCollector.php

```diff
@@ -15,8 +15,14 @@ public function registerEventListeners(): void
         $startTime = defined('LARAVEL_START') ? LARAVEL_START : microtime(true);
         $this->addSegment('laravel boot', $startTime);
 
-        $this->app->booted(function () {
-            $this->endSegment('laravel boot');
-        });
+        if($this->app instanceof \Laravel\Lumen\Application){
+                $this->endSegment('laravel boot');
+        }
```

Comment:
How did you test this logic? It seems incorrect as you have two endsection and incorrect type.